pkg/types/types.go

```diff
@@ -80,6 +80,7 @@ type NetConf struct {
 	} `json:"ipam,omitempty"`
 	Args                 Args                   `json:"args"`
 	MTU                  int                    `json:"mtu"`
+	NumQueues            int                    `json:"n_queues"`
```

Comment:
One of my colleagues has asked if we can change this JSON name to "num_queues", for a little more immediate clarity.